src/services/resource.js

```diff
@@ -1178,4 +1182,69 @@ module.exports = class resourceHelper {
 
 		return orgConfigList[resourceType]
 	}
+
+	/**
+	 * Get resources from consumption service
+	 * @name browseExistingList
+	 * @param {string} organization_id - Org Id of the user
+	 * @param {string} token - Token of the user
+	 * @param {Object} query - Query object passed by user
+	 * @param {string} searchText - Title to search
+	 * @param {Integer} pageNo -  Used to skip to different pages. Used for pagination . If value is not passed, by default it will be 1
+	 * @param {Integer} pageSize -  Used to limit the data. Used for pagination . If value is not passed, by default it will be 100
+	 * @returns {Object} - Response contain object of user details
+	 */
+	static async browseExistingList(organization_id, token, query, searchText = '', pageNo, pageSize) {
+		try {
+			let result = {
+				data: [],
+				count: 0,
+			}
+			const type = query[common.TYPE] ? query[common.TYPE] : ''
+			const search = searchText != '' ? searchText : ''
```

Comment:
@adithyadinesh0412 if the consumption service is self then we dont need to fetch the data from consumption side